AutomatticTracks/src/main/java/com/automattic/android/tracks/MessageBuilder.java

```diff
@@ -29,7 +30,7 @@
     public static final String ALIAS_USER_ANONID_PROP_NAME = "anonId";
 
     public static synchronized boolean isReservedKeyword(String keyToTest) {
-        String keyToTestLowercase = keyToTest.toLowerCase();
+        String keyToTestLowercase = keyToTest.toLowerCase(Locale.ROOT);
```

Comment:
Docs on `Locale.ROOT`:
> Useful constant for the root locale.  The root locale is the locale whose
    language, country, and variant are empty ("") strings.  This is regarded
    as the base locale of all locales, and is used as the language/country
    neutral locale for the locale sensitive operations.

In this context is a good choice as those data-related operations are not related to any real `Locale`.